index.html

```diff
@@ -15,224 +16,268 @@
+              <rect y="0.455811" width="18" height="18" rx="9" fill="#FF6B00"/>
+            </svg>     
+            <h2>Skills</h2>
+          </div>
+          <div>
             <ul>
               <li>Codekit</li>
               <li>Github</li>
               <li>Codepen</li>
               <li>Gitlab</li>
               <li>Terminal</li>
             </ul>
-          </div>         
- 
-
-    </section>
-    <!-- contact-form section. -->
-    <section class="contact-section">
-            <p class="contact-head">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-            <form class="form" action="https://formspree.io/f/mvonaqyb" method="POST">
-              <div>
-                <input class="form-input" type="text" name="name" placeholder="First name" maxlength="30" required>
-                <hr>
-              </div>
-              <div>
-                <input class="form-input" type="email" name="email" placeholder="Email address" maxlength="30" required>
-                <hr>
-              </div>
-                <textarea name="mail-message" id="" cols="20" rows="5" placeholder="Enter text here" maxlength="100%" required style="border: none;"></textarea>
-                <input class="send" type="submit" value="Get in touch">
-            </form>
-    </section>
-    <footer>
-        <hr>
-          <ul class="footer-container">
-            <li><a><img src="images/github2.png" alt="github"/></a></li>
-            <li><a><img src="images/Icon - Linkedin.png" alt="linkedin"/></a></li>
-            <li><a><img src="images/Icon - Angelist.png" alt="angel-list"/></a></li>
-            <li><a><img src="images/Icon - Twitter.png" alt="twitter"/></a></li>
-            <li><a><img src="images/Icon - Medium.png" alt="medium"/></a></li>
-          </ul>
-          <div class="ending-line">
-            
           </div>
-      </footer>
+        </div>
+      </div>
+      </section> 
 
-   
-
-         
-   
-    
-</body>
+      <section class="contact-section">
+        <p class="contact-head">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        <form class="form" action="https://formspree.io/f/mvonaqyb" method="POST">
+          <div>
+            <input class="form-input" type="text" name="name" placeholder="First name" maxlength="30" required>
+            <input class="form-input" type="text" name="name" placeholder="Last name" maxlength="30" required>
+          </div>
+            <input class="form-input" type="email" name="email" placeholder="Email address" maxlength="30" required>
+            <textarea name="mail-message" id="" cols="20" rows="5" placeholder="Enter text here" maxlength="500" required></textarea>
+            <input class="send" type="submit" value="Get in touch">
+        </form>
+      </section>
```

Comment:
- [OPTIONAL] The same thing happened here too. Your design looks different to how it appears on the figma design. Kindly work your design alongside the figma design. I made it optional because it was not mentioned by the previous reviewer. So, I recommend you attend to it.

### Your Design
![your-contact-me](https://user-images.githubusercontent.com/26597432/222907102-b7e5d954-60dd-4d9d-99cc-5ffaac088c94.png)

### Figma Design 
![temp-contact-me](https://user-images.githubusercontent.com/26597432/222907118-95f8dacb-4166-4901-a496-db331b4abaf7.png)